### PR TITLE
[Refactor] #391 Remove unused code in network interacting

### PR DESCRIPTION
**Pull request by MisterVants**
Self-explanatory

### CODE REVIEW

File: Sources/WalletConnectSign/Engine/Common/ApproveEngine.swift [hunk 1]
```diff
@@ -140,7 +140,7 @@ final class ApproveEngine {
         try await networkingInteractor.subscribe(topic: topic)
         sessionStore.setSession(session)
 
-        try await networkingInteractor.request(.wcSessionSettle(settleParams), onTopic: topic)
+        networkingInteractor.request(.wcSessionSettle(settleParams), onTopic: topic)
```
Comment (flypaper0) [inline Sources/WalletConnectSign/Engine/Common/ApproveEngine.swift:143]: Why async call removed?

File: Sources/WalletConnectSign/NetworkInteractor/NetworkInteractor.swift [hunk 2]
```diff
@@ -62,18 +59,6 @@ class NetworkInteractor: NetworkInteracting {
         setUpPublishers()
     }
 
-    func request(_ wcMethod: WCMethod, onTopic topic: String) async throws {
```
Comment (flypaper0) [inline Sources/WalletConnectSign/NetworkInteractor/NetworkInteractor.swift:65]: the idea was replace `request` method with async `request`. But here you removed all async calls

**Comment by MisterVants** [comment]
Removing the async method makes it impossible to handle errors beyond the interactor call. Changing it to draft.

File: Sources/WalletConnectSign/NetworkInteractor/NetworkInteractor.swift (hunk 2 above)
Comment (llbartekll) [reply Sources/WalletConnectSign/NetworkInteractor/NetworkInteractor.swift:65]: totally agree, do not get the purpose of doing so.

**Review by llbartekll — COMMENTED** [review]
Can we close it?

**Comment by MisterVants** [comment]
> Can we close it?

yes

**Pull request closed without merging**